Sources/ProjectDescription/Target.swift

```diff
@@ -41,6 +41,9 @@ public struct Target: Codable, Equatable {
     /// Relative paths to the resources directory.
     public let resources: [FileElement]?
 
+    /// Copy files phases.
```

Comment:
I would not call it a phase, something like `Files to be bundled with application` would be better, I think

Reply:
I've added a comment with more context, see if that makes sense for you...

Reply:
But agreed that this need better clarification for documentation purpose.